model/user.go

```
@@ -24,6 +25,17 @@ type User struct {
 	IsDisabled bool `json:"is_disabled" gorm:"comment:是否禁用"`
 }
 
+func (u *User) BeforeCreate(tx *gorm.DB) (err error) {
+	if u.Password != "" {
+		if err := u.HashPassword(); err != nil {
+			return err
+		}
+	}
+
+	u.BaseModel.BeforeCreate(tx)
```

golangci-lint

Error return value of `u.BaseModel.BeforeCreate` is not checked

(errcheck)